### DISCUSSION TITLE
Custom text with URL in footer

### DISCUSSION THREAD

**Discussion by m3xw3ll** [danny-avila/LibreChat#2310] [Help Wanted]
Hello all,
there is the possibility to create a custom footer in the `.env` file. There is also a merged [PR](https://github.com/danny-avila/LibreChat/pull/2111) to add custom ToS and Privacy Policy to the footer. 
I tried to add those in the `.env` file like so:
`CUSTOM_FOOTER="<html><a href='www.google.de' target='_blank'>Hallo</a></html>"`

But this just shows the plain string not proper formatted. I tried different approaches like removed the HTML tag, but no success. Can you please tell me how to achieve this?

**Comment by fuegovic** [comment]
you can't currently add a link to the footer without editing the code yourself

**Reply by m3xw3ll** [reply]
And where can I change this in the code?

**Reply by fuegovic** [reply]
I don't remember the exact place but you should be able to find it easily if you search the project for "footer" in vscode

**Comment by TobiasGoerke** [comment]
The `CUSTOM_FOOTER` supports markdown, i.e.:
`CUSTOM_FOOTER=[Google](https://google.com)`
